standard_schema/hedwiki/HED8.2.0.mediawiki

```diff
@@ -196,41 +196,47 @@ Each term in this vocabulary has a human-readable description and may include ad
 ****** Nose <nowiki>[A structure of special sense serving as an organ of the sense of smell and as an entrance to the respiratory tract.]</nowiki>
 ****** Mouth <nowiki>[The proximal portion of the digestive tract, containing the oral cavity and bounded by the oral opening.]</nowiki>
-****** Teeth <nowiki>[The hard bonelike structures in the jaws. A collection of teeth arranged in some pattern in the mouth or other part of the body.]</nowiki>
-**** Lower-extremity <nowiki>[Refers to the whole inferior limb (leg and/or foot).]</nowiki>
-***** Ankle <nowiki>[A gliding joint between the distal ends of the tibia and fibula and the proximal end of the talus.]</nowiki>
-***** Calf <nowiki>[The fleshy part at the back of the leg below the knee.]</nowiki>
-***** Foot <nowiki>[The structure found below the ankle joint required for locomotion.]</nowiki>
-****** Big-toe <nowiki>[The largest toe on the inner side of the foot.]</nowiki>
-****** Heel <nowiki>[The back of the foot below the ankle.]</nowiki>
-****** Instep <nowiki>[The part of the foot between the ball and the heel on the inner side.]</nowiki>
-****** Little-toe <nowiki>[The smallest toe located on the outer side of the foot.]</nowiki>
-****** Toes <nowiki>[The terminal digits of the foot.]</nowiki>
-***** Knee <nowiki>[A joint connecting the lower part of the femur with the upper part of the tibia.]</nowiki>
-***** Shin <nowiki>[Front part of the leg below the knee.]</nowiki>
-***** Thigh <nowiki>[Upper part of the leg between hip and knee.]</nowiki>
+******* Teeth <nowiki>[The hard bonelike structures in the jaws. A collection of teeth arranged in some pattern in the mouth or other part of the body.]</nowiki>
+**** Neck <nowiki>[The part of the body connecting the head to the torso, containing the cervical spine and vital pathways of nerves, blood vessels, and the airway.]</nowiki>
 **** Torso <nowiki>[The body excluding the head and neck and limbs.]</nowiki>
-***** Torso-back <nowiki>[The rear surface of the human body from the shoulders to the hips.]</nowiki>
-***** Buttocks <nowiki>[The round fleshy parts that form the lower rear area of a human trunk.]</nowiki>
 ***** Torso-chest <nowiki>[The anterior side of the thorax from the neck to the abdomen.]</nowiki>
-***** Gentalia <nowiki>{deprecatedFrom=8.1.0}[The external organs of reproduction.]</nowiki>
-***** Hip <nowiki>[The lateral prominence of the pelvis from the waist to the thigh.]</nowiki>
-***** Waist <nowiki>[The abdominal circumference at the navel.]</nowiki>
+***** Torso-back <nowiki>[The rear surface of the human body from the shoulders to the hips, often regarded as the back]</nowiki>
+***** Abdomen <nowiki>[The part of the body between the thorax and the pelvis.]</nowiki>
+***** Pelvis <nowiki>[The bony structure at the base of the spine supporting the legs.]</nowiki>
+****** Buttocks <nowiki>[The round fleshy parts that form the lower rear area of a human trunk.]</nowiki>

```

Comment:
I don't think Buttocks should go under Pelvis. It should be at the same level.  It is possible that we need to do:

**** Torso  
**** Torso-part  
***** Buttocks  

Now that we are going to ontologies -- a strict is-a will be enforced.

Reply:
Teeth is in the Mouth (cavity). The same way that is under the Head hierarchy?